tests/es-bulk-retry.sh

```diff
@@ -77,9 +78,8 @@ if $msg contains "msgnum:" then {
 	call try_es
 }
 
-action(type="omfile" file=`echo $RSYSLOG_OUT_LOG`)
+action(type="omfile" file="'$RSYSLOG_OUT_LOG'")
```

Comment:
this does not work, it needs to be as in the original line. We use the backticks feature here: https://www.rsyslog.com/doc/v8-stable/rainerscript/constant_strings.html

I recently added the env var references so that we in the medium term can generate test-specific names and thus support parallel execution.

Reply:
I thought this may be using shell escape (double quotes), but it isn't . so the file name will actually be '$RSYSLOG_OUT_LOG'. I would expect that CI fails.

Reply:
my bad, got lost in the quotes myself ;-) Albeit I would still prefer the previous code, as it would be consistent with the other tests. But definitely no hard need to change.

Reply:
on second thought, your way of doing things I think is actually better. ;-) Let's keep it, I'll see if I sed the rest of the testbench. ;-)

Reply:
re: quotes - I'm really surprised that
```
`echo $RSYSLOG_OUT_LOG`
```
worked at all - I would have thought that the backticks would have been preserved and not expanded inside the single quotes

re: uuid - I changed the test not to have that dependency, and added a note that it is recommended to use `$uuid` in production